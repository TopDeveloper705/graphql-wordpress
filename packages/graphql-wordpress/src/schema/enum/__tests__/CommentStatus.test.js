import COMMENT_STATUS from 'enum/CommentStatus';

describe('Test enum values', () => {
  test('COMMENT_STATUS values', () => {
    expect(COMMENT_STATUS.getValues()).toMatchSnapshot();
  });
});
