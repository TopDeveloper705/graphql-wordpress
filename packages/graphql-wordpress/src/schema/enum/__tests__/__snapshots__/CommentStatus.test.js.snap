// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values COMMENT_STATUS values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "OPEN",
    "value": "open",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "CLOSED",
    "value": "closed",
  },
]
`;
